Healthy/Classes/Modules/SavedRecipes/Cell/SavedRecipesTableViewCell.swift

```diff
@@ -36,15 +36,33 @@ final class SavedRecipesTableViewCell: UITableViewCell {
         configureLayout()
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10))
+    }
+
     // MARK: Configure
 
-    func update(with viewModel: ViewModel) {
+    func update(with viewModel: SavedRecipe) {
         // TODO: Set recipe imageView using kingfisher and set (patternFood) as placeholder.
-        titleLabel.text = viewModel.title
-        chefNameLabel.text = viewModel.chefName
-        timeLabel.text = viewModel.time
+        if let image = viewModel.recipeImage {
+            recipeImageView.image = image
+        }
+
+        if let title = viewModel.title {
+            titleLabel.text = title
+        }
+
+        if let chefName = viewModel.chefName {
+            chefNameLabel.text = chefName
+        }
+
+        if let cookingTime = viewModel.cookingTime {
+            timeLabel.text = "\(cookingTime) min"
+        }
```

Comment:
```suggestion
        recipeImageView.image = viewModel.recipeImage
        titleLabel.text = viewModel.title
        chefNameLabel.text = viewModel.chefName
        
        if let cookingTime = viewModel.cookingTime {
            timeLabel.text = "\(cookingTime) min"
        } else {
            timeLabel.text = nil        
        }
```

Text and image are optional. Optional chaining.